app/assets/javascripts/tutorial.js

```diff
@@ -26,3 +26,7 @@ function onPlayerStateChange(event) {
     message.innerHTML = "You watched them all. Bask in the glory of your new skills."
   }
 }
+
+$(document).ready(function(){
+  $('[data-toggle="tooltip"]').tooltip();
+});
```

Comment:
Interesting approach! I would love to hear more about what this does during our slack call tomorrow. Nice work!